package.json

```diff
@@ -22,8 +22,8 @@
     "@types/angular-animate": "^1.5.10",
     "@types/angular-mocks": "^1.7.0",
     "@types/ckeditor": "^4.9.2",
-    "@types/d3": "^3.5.40",
-    "@types/google.visualization": "0.0.46",
+    "@types/d3": "^5.7.2",
```

Comment:
I see 5.7.2 here and 5.9.2 above, is this expected?

Reply:
Ok, I realized the latest version of types available for d3 is 5.7.2. (Marking resolved)